src/sonic-eventd/tools/rsyslog_volume_test.py

```diff
@@ -0,0 +1,61 @@
+import sys
+import subprocess
+import time
+import logging
+import argparse
+
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    handlers = [
+        logging.FileHandler("debug.log"),
+        logging.StreamHandler(sys.stdout)
+    ]
+)
+
+def read_events_from_file(count):
+    logging.info("Reading from file generated by events_tool")
+    lines = 0
+    with open('STDOUT', 'r') as file:
```

Comment:
The STDOUT being used as file name is tool's bug, which I should fix. So if user would not give any file, I should send it to STDOUT, not file "STDOUT"

Can you please explicitly pass a file name ?
If not this  code will be broken, when I fix the events_tool.

Reply:
Done